proto/sbom/sbom.proto

```diff
@@ -19,8 +19,9 @@ message SBOMPayload {
 message SBOMEntity {
   SBOMSourceType                     type               = 1;
   string                             id                 = 2; // Unique identifier to be able to correlated and "deduplicate" SBOM
+  repeated string                    dd_tags            = 7; // datadog tags that will be added by the agent depending of the SBOMSourceType
   optional google.protobuf.Timestamp generatedAt        = 3; // the datetime of the SBOM generation
-  repeated string                    tags               = 4; // datadog tags that will be added by the agent depending of the SBOMSourceType
+  repeated string                    repo_tags          = 4; // the tags of the container image
```

Comment:
Yes.
Renaming the field will require a change on the source code (done here: https://github.com/DataDog/datadog-agent/pull/15899/files#diff-50e6eb334ab3ef40500c8d556ef29302bbc6ad4c5b604138ed069c7e0e5fb642R122), but it doesn’t impact the message on the wire, as long as the ID doesn’t change, so that the recipient can still use the previous message definition.

See
* https://stackoverflow.com/questions/45431685/protocol-buffer-does-changing-field-name-break-the-message
* https://groups.google.com/g/protobuf/c/hX4Mj0P4N0w